### ISSUE TITLE
Wrong Breadcrumb for Customer Doctype

### ISSUE THREAD

**Issue by trustedcomputer** [frappe/erpnext#34067]
### Information about bug

Customer Doctype should be under selling, not retail:

$ ls -l ~/frappe-bench/apps/erpnext/erpnext/selling/doctype/
total 0
-rw-r--r-- 1 frappe frappe   0 Feb 10 19:03 __init__.py
drwxr-xr-x 1 frappe frappe  46 Feb 10 21:10 __pycache__
drwxr-xr-x 1 frappe frappe 218 Feb 10 19:03 campaign
drwxr-xr-x 1 frappe frappe 288 Feb 15 02:03 customer
drwxr-xr-x 1 frappe frappe 144 Feb 14 03:30 customer_credit_limit
.....
![Customer](https://user-images.githubusercontent.com/75872475/218911152-db676b1d-e1ac-4253-a6f5-fc1c94ce0cfb.png)

### Module

selling

### Version

$ bench version
erpnext 13.45.1
frappe 13.49.1

### Installation method

docker

**Comment by dj12djdjs** [comment]
Assigning Customer doctype to Retail module doesn't make sense. If company doesn't operate in Retail space. It looks strange. Selling module I think is better name for Customer doctype.

**Comment by trustedcomputer** [comment]
I just noticed the issue is more general than just the Customer Doctype. Also, Quotations and Sales Orders are affected. Maybe more?

![Screenshot 2023-02-16 at 08-11-41 Sales Order](https://user-images.githubusercontent.com/75872475/219423396-094e86ad-88c6-455b-b1e1-f7c16c5c6aac.png)
![Screenshot 2023-02-16 at 08-11-18 Quotation](https://user-images.githubusercontent.com/75872475/219423433-8709519c-14fe-4c6b-b58f-6f685c61baba.png)

**Comment by s-aga-r** [comment]
There is no module named `Retail` in ERPNext, somehow the `Selling` module is represented as `Retail`. @deepeshgarg007 ?

ref: https://github.com/frappe/erpnext/blob/develop/erpnext/modules.txt

**Comment by trustedcomputer** [comment]
Retail is a domain:
![image](https://user-images.githubusercontent.com/75872475/220178942-50fd03ae-2b4c-41b8-86df-841d84ab6e82.png)

**Comment by trustedcomputer** [comment]
@dj12djdjs @s-aga-r 
I have narrowed this down to somewhere in the ERPNext release 13.45.0. 

In a freshly installed 13.44.0, this issue does not appear. The doctypes Customer, Quotation, Sales Order use the Selling breadcrumb. 
In a freshly installed 13.45.0 (the very next release), those same doctypes all use the Retail breadcrumb.
If a freshly installed 13.44.0 is upgraded to 13.45.0, those doctypes begin using the Retail breadcrumb (but only after "bench migrate"). 
If that upgraded 13.45.0 instance is downgraded to 13.44.0, those doctypes still use the Retail breadcrumb (even after "bench migrate").

These tests were all done with frappe_docker, if that makes any difference.

**Comment by trustedcomputer** [comment]
@dj12djdjs @s-aga-r 
I have found a related CLOSED issue with a workaround here: https://github.com/frappe/erpnext/issues/32609#issuecomment-1280725872

I am not sure why this works, but it does. "Selling" is the module for the "Retail" workspace both before and after installing v13.45.0. So it still doesn't make sense to me, but it works.

So two workarounds I have found that work on a test install of ERPNext v13.45.0 are:
1. Delete the Retail workspace altogether
2. Go into the mariadb database and manually update the value of the "module" column for "Retail" in the "tabWorkspace" table to null: 

`UPDATE tabWorkspace SET module = '' WHERE name = 'Retail';`

These are only workarounds. But I think with this information, someone might be able to find the real root cause?
